clouddrift/adapters/gdp.py

```diff
@@ -19,7 +19,9 @@
 
 GDP_VERSION = "2.00"
 GDP_DATA_URL = "https://www.aoml.noaa.gov/ftp/pub/phod/lumpkin/hourly/v2.00/netcdf/"
-GDP_FILENAME_PATTERN = "drifter_{id}.nc"
+GDP_DATA_URL_EXPERIMENTAL = (
```

Comment:
I forgot where (I believe on the Pangeo JupyterHub) FTP links are blocked for security reasons. So I would suggest we switch to the https:// URL pointing to the same location: https://www.aoml.noaa.gov/ftp/pub/phod/lumpkin/hourly/experimental/.